### ISSUE TITLE
Mix-up in tags when embedding content from another page

### ISSUE THREAD

**Issue by R-Quancard** [stevencohn/OneMore#1736]
# Problem to Solve

When I embed content from a reference page into another page (let's call it "container page"), if the reference content includes tags, the embedded section is displayed with mixed-up tags. Actually, it seems that the tags in the embedded contents are replaced by the first ones that appear in the container page.

# Screenshots

See below an example reference page contents, with the tags as originally chosen:
![image](https://github.com/user-attachments/assets/9f7f0914-f3e4-424a-aac5-d316b79fa242)

See below an example container page with embedded contents directly from the above reference page:
![image](https://github.com/user-attachments/assets/2557e435-effa-4a55-be37-5c9c5ef02e93)

We see that the checkboxes are replaced by the "home" tag, which is the first one used in the container page; the yellow highlighting is replaced by the link tag, which is the second one used in the container page; and the red highlighting is replaced by the yellow, which is used in third position in the container page.

# Environment

 - OneNote Version: Microsoft® OneNote® for Microsoft 365 MSO (Version 2411 Build 16.0.18227.20082) 64-bit
 - OneMore Version 6.6.1

... and thanks for this great add-in!

**Comment by stevencohn** [comment]
Hi @R-Quancard . Ah, OK! I was not able to reproduce that aspect of the ticket. I hadn't realized you were highlighting via the tags themselves. I can understand why.  FYI, it should work if you do use the normal text highlight button on the Home tab.

**Comment by R-Quancard** [comment]
Yes, that's true. Highlighting via the tags allows me to toggle more efficiently between yellow / red / no highlighting, and it allows me to research those tagged items as well. Of course if you find a way and have time to correct this it at some point, it will be greatly appreciated! But if things stay this way I can use a workaround indeed. Regardless, thanks for this embedding feature which is very useful!

**Comment by stevencohn** [comment]
https://github.com/stevencohn/OneMore/releases/tag/6.7.1
